### ISSUE TITLE
Image import tag issue

### ISSUE THREAD

**Issue by zszeron** [containerd/nerdctl#3701]
### Description

I transferred the compressed package to another server using `nerdctl save image:latest | gzip > images.tar.gz`. There was a tag issue. What could be the reason?
```shell
[root@master ~]# nerdctl images | grep registry.k8s.io
registry.k8s.io/ingress-nginx/kube-webhook-certgen    v1.4.4                  a9f03b34a3cb    28 seconds ago    linux/amd64    66.59MB    26.15MB
registry.k8s.io/ingress-nginx/controller              v1.12.0-beta.0          9724476b9289    37 seconds ago    linux/amd64    296.2MB    104.6MB
[root@master ~]# crictl images | grep registry.k8s.io
registry.k8s.io/ingress-nginx/controller:v1.12.0-beta.0     <none>                 11b916a025f02       105MB
registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.4.4   <none>                 a62eeff05ba51       26.2MB
```

### Steps to reproduce the issue

1.
2.
3.

### Describe the results you received and expected

crictl  and nerdctl   agreement

### What version of nerdctl are you using?

nerdctl version 2.0.0

### Are you using a variant of nerdctl? (e.g., Rancher Desktop)

None

**Comment by fengwei0328** [comment]
[#3702](https://github.com/containerd/nerdctl/issues/3702)
https://github.com/containerd/containerd/issues/11000
Maybe it will be able to explain your doubts and solve your problems

**Comment by apostasie** [comment]
@AkihiroSuda there has been a few cri (plugin) issues lately (#3730, #3671).

Where do we stand with this?

As I see it from a distance, cri plugin & containerd are just not playing nice with each other, and I am wondering if this is worth the effort or not.

If we do not want to do this, maybe we could document it, and close these issues - or if we want to support it down the road, maybe have a central issue to collect / direct peeps to.

Thoughts?

**Comment by AkihiroSuda** [comment]
> There was a tag issue. 

What issue?

**Comment by apostasie** [comment]
> > There was a tag issue.
> 
> What issue?

I assume OP is talking about the fact that tags are missing when viewed with `crictl images` since (again, AFAIK) crictl relies on its own separate meta.db instead of querying containerd.

**Comment by fengwei0328** [comment]
> > > There was a tag issue.
> > 
> > 
> > What issue?
> 
> I assume OP is talking about the fact that tags are missing when viewed with `crictl images` since (again, AFAIK) crictl relies on its own separate meta.db instead of querying containerd.

Yes, the reason for the display has been found. I am wondering if `nerdctl` can implement some special handling for the `k8s.io` namespace to ensure that `nerdctl -n k8s.io images` only shows one entry, and when deleting images, it can delete all three at once.

**Comment by fahedouch** [comment]
> > > > There was a tag issue.
> > > 
> > > 
> > > What issue?
> > 
> > 
> > I assume OP is talking about the fact that tags are missing when viewed with `crictl images` since (again, AFAIK) crictl relies on its own separate meta.db instead of querying containerd.
> 
> Yes, the reason for the display has been found. I am wondering if `nerdctl` can implement some special handling for the `k8s.io` namespace to ensure that `nerdctl -n k8s.io images` only shows one entry, and when deleting images, it can delete all three at once.

The example includes a tag, but it's not formatted in the nerdctl style, which seems reasonable.

**Comment by fengwei0328** [comment]
> > > > > There was a tag issue.
> > > > 
> > > > 
> > > > What issue?
> > > 
> > > 
> > > I assume OP is talking about the fact that tags are missing when viewed with `crictl images` since (again, AFAIK) crictl relies on its own separate meta.db instead of querying containerd.
> > 
> > 
> > Yes, the reason for the display has been found. I am wondering if `nerdctl` can implement some special handling for the `k8s.io` namespace to ensure that `nerdctl -n k8s.io images` only shows one entry, and when deleting images, it can delete all three at once.
> 
> The example includes a tag, but it's not formatted in the nerdctl style, which seems reasonable.

```
nerdctl -n k8s.io images --names --format wide
NAME                                                                                                DIGEST                                                                     IMAGE ID        CREATED          PLATFORM       SIZE         BLOB SIZE
docker.io/library/centos:7                                                                          sha256:be65f488b7764ad3638f236b7b515b3678369a5124c47b8d32916d6487418ea4    be65f488b776    3 minutes ago    linux/amd64    211.5 MiB    72.6 MiB
docker.io/library/centos@sha256:be65f488b7764ad3638f236b7b515b3678369a5124c47b8d32916d6487418ea4    sha256:be65f488b7764ad3638f236b7b515b3678369a5124c47b8d32916d6487418ea4    be65f488b776    3 minutes ago    linux/amd64    211.5 MiB    72.6 MiB
sha256:eeb6ee3f44bd0b5103bb561b4c16bcb82328cfe5809ab675bb17ab3a16c517c9                             sha256:be65f488b7764ad3638f236b7b515b3678369a5124c47b8d32916d6487418ea4    be65f488b776    3 minutes ago    linux/amd64    211.5 MiB    72.6 MiB
```
Regarding the output above, how can I align with `crictl` to display only one entry?
```
$ crictl images       
IMAGE                      TAG                 IMAGE ID            SIZE
docker.io/library/centos   7                   eeb6ee3f44bd0       76.1MB
```
